### ISSUE TITLE
Missing/transparent color on Tilix title

### ISSUE THREAD

**Issue by curtisbelt** [mythio/macterial#8]
Hello! First off, great theme thank you :).

So on my terminal ([Tilix](https://github.com/gnunn1/tilix)), there is this bar that is transparent on your theme, but not others. I am not sure how tell you "which color setting" that is as far as your theme's code goes. Please let me know if there's anything else I can provide.

![image](https://user-images.githubusercontent.com/4251760/57391650-8f946680-7184-11e9-9a1e-d22364f5ffc4.png)

Here it is with Adwaita. Also tried Arc, Qogir which I have previously used. All of them have this bar colored.

![image](https://user-images.githubusercontent.com/4251760/57391782-cbc7c700-7184-11e9-9791-9586a63759e0.png)

```
OS: Fedora release 30 (Thirty) x86_64 
Kernel: 5.0.11-300.fc30.x86_64 
Uptime: 5 hours, 17 mins 
Packages: 2090 (rpm), 8 (snap) 
Shell: zsh 5.7.1 
Resolution: 2560x1080, 2560x1080, 2560x1080, 2560x1080 
DE: GNOME 3.32.1 
Theme: macterial [GTK2/3] 
Icons: Papirus [GTK2/3] 
Terminal: tilix 
CPU: Intel i7-7700K (8) @ 4.500GHz 
GPU: AMD ATI Radeon VII 
Memory: 13048MiB / 31965MiB 
```

**Comment by ghost** [comment]
Yes there have been many such issues after I performed code cleanup. Will refactor the code base again in the next release.
